src/renderers/shaders/ShaderChunk/global_fragment.glsl

```diff
@@ -0,0 +1 @@
+// template for injecting varying, uniform and functions above main below other includes
```

Comment:
dunno, maybe, im looking for feedback, and weighting in all the input so far, have to admit didn't know there were pull requests already made for this

can you define abuse more? The current chunks are what we have to work with now, if we don't have a meaningful and convenient hook right now, why do you think that this is abuse, and not a start of something like that?